models/_sector/dex/trades/avalanche_c/_sources.yml

```diff
@@ -2,6 +2,10 @@ version: 2
 
 sources:
   - name: uniswap_v3_avalanche_c
+  - name: pharaoh_v1_avalanche_c
+    tables:
+      - name: ClPool_evt_Swap
+      - name: ClPoolFactory_evt_PoolCreated
```

Comment:
Since the wrong sources added in this PR,
I think I will squeeze the changed files here into #5068 .